core/pom.xml

```diff
@@ -128,8 +128,8 @@
         </dependency>
 
         <dependency>
-            <groupId>org.easymock</groupId>
-            <artifactId>easymock</artifactId>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
```

Comment:
This makes no sense - don't add a dependency that is never used. If a subsequent change will require this dependency, the onus will be on them.

Reply:
I agree with @rhusar - the org.easymock dependency should be removed if it is not used at all.